lib/chezmoi/fsmutator_windows.go

```diff
@@ -0,0 +1,148 @@
+// +build windows
+
+package chezmoi
+
+import (
+	"os"
+	"path/filepath"
+	"syscall"
+
+	acl "github.com/hectane/go-acl"
+
+	"github.com/google/renameio"
+	"golang.org/x/sys/windows"
+
+	vfs "github.com/twpayne/go-vfs"
+	"github.com/twpayne/go-vfs/vfst"
+)
+
+func (a *FSMutator) getCorrectedPath(file string) string {
```

Comment:
I don't know if there's a better way to deal with the fact that during the tests a path will get passed in that needs to have the test sandbox dir prepended to it.  This feels like a hack but it's probably better than pushing IsPrivate up into `go-vfs`.